frontend/app/routes/$lang+/_protected+/data-unavailable.tsx

```diff
@@ -40,10 +41,12 @@ export async function loader({ request }: LoaderFunctionArgs) {
 export default function DataUnavailable() {
   const { t } = useTranslation(handle.i18nNamespaces);
   const { SCCH_BASE_URI } = useLoaderData<typeof loader>();
+  const origin = useUserOrigin();
+
   const doyouqualify = <InlineLink to={t('do-you-qualify.href')} />;
   const howtoapply = <InlineLink to={t('how-to-apply.href')} />;
   const contactus = <InlineLink to={t('contact-us.href', { baseUri: SCCH_BASE_URI })} />;
-  const mydashboard = <InlineLink to={t('my-dashboard.href', { baseUri: SCCH_BASE_URI })} />;
+  const mydashboard = <InlineLink to={origin?.to ?? ''} />;
```

Comment:
Similar comment to the `DropdownMenuItem` comment, I feel we shouldn't render a broken link at all if the `origin` isn't found.

Reply:
removed